modules/ROOT/pages/policies-compare-mule3-and-mule4.adoc

```diff
@@ -0,0 +1,40 @@
+= Policies and Mule Versions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+In Mule versions 3 and 4, the policies remain the same, with a few exceptions. The following section describes the similarities and differences in these Mule versions.
+
+// Policies in Mule 4
+// Classloader isolation exists between the application, the runtime and connectors, and policies. To fulfill the isolation requirement, policies are self-contained and packaged to include libraries needed for execution, similar to an application. Additionally, certain changes to the packaging, size, and scope of a policy are introduced in Mule 4.
+
+[%header,cols="40a,50a,50a"]
+|===
+| Feature | Mule 3 | Mule 4 
+|           | Uses MEL expressions | Uses DataWeave 2.0 
+| Policies distribution | Policies cannot be distributed outside of runtime | Policies can be distributed outside of the runtime, which simplifies upgrades 
+| Policies  | Partial non-blocking processing implemented for policies | * All policies are non-blocking to improve performance
```

Comment:
We can replace it for 

| Policies performance | Partial non-blocking processing implemented for policies | All policies implement a non-blocking processing strategy |

| Versioning | Policies don't support proper versioning | Versioning is supported and enforced |

Reply:
Done! Thanks!